Products/CMFPlone/PloneTool.py

```diff
@@ -913,7 +915,7 @@ def listMetaTags(self, context):
         return result
 
     @security.public
-    def getUserFriendlyTypes(self, typesList=None):
+    def getUserFriendlyTypes(self, typesList=[]):
```

Comment:
No no no, don't do this. Classic error:

```
>>> def foo(bar=[]):
...     bar.append(1)
...     print(bar)
... 
>>> foo()
[1]
>>> foo()
[1, 1]
>>> foo()
[1, 1, 1]
>>> foo()
[1, 1, 1, 1]
```

Same remark is true for `plone.base` where you copied this to.

Reply:
Sorry, in fact I know this.... but brain-malfunction.